WordPress-iOS-Shared/Core/WPStyleGuide.m

```diff
@@ -141,6 +141,11 @@ + (UIFont *)tableviewSectionHeaderFont
     return [WPFontManager systemRegularFontOfSize:13.0];
 }
 
++ (UIFont *)tableviewSectionFooterFont
+{
+	return [WPFontManager systemRegularFontOfSize:13.0];
```

Comment:
This was indeed done for consistency, but with the default size that appears for `UITableView` instead.